upup/models/cloudup/resources/addons/networking.weave/v1.9.4.yaml

```diff
@@ -0,0 +1,118 @@
+kind: ClusterRole
```

Comment:
hello @chrislovecnm 
added switch for different versions of k8s. old(1.5) will continue use weave 1.9.3